service/src/main/java/bio/terra/cbas/controllers/MethodsApiController.java

```diff
@@ -453,6 +494,16 @@ private void addLastRunDetails(List<MethodDetails> methodDetails) {
 
   private MethodDetails methodToMethodDetails(Method method, boolean includeVersions) {
 
+    Boolean isMethodPrivate = false;
+
+    if (Objects.equals(
+        method.methodSource(), PostMethodRequest.MethodSourceEnum.GITHUB.toString())) {
+      if (githubMethodDetailsDao.getMethodSourceDetails(method.methodId()) != null) {
+        isMethodPrivate =
+            githubMethodDetailsDao.getMethodSourceDetails(method.methodId()).isPrivate();
```

Comment:
nit: if GitHub method has data in `github_method_details` table, here it would end up running the same query in   `getMethodSourceDetails()` twice. To avoid this you could store that value in a variable and use it on L503 instead of re-running the query.